test/extended/networking/multus.go

```diff
@@ -0,0 +1,118 @@
+		makeNamespaceScheduleToAllNodes(f)
+		ns = f.Namespace.Name
+		fmt.Println()
+		o.Expect(testMultus(f, oc, ns)).To(o.Succeed())
+	})
+})
+
+/*
+	Desc: 	Multus is already installed on origin. This test aims to verify the integrity of the installation.
+		  	It follows the steps listed in the multus quickstart guide.
+			(See: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/quickstart.md#storing-a-configuration-as-a-custom-resource)
+	Overview: Creates a net-attach-def, creates a pod with an annotation that refers to it,
+			 and checks for newly created net1 interface.
+*/
+func testMultus(f *e2e.Framework, oc *exutil.CLI, ns string) error {
+	// Setup
+	var err error
+	podName := "multus-test-pod"
+
+	expectNoError(err)
+
+	g.By("attempting to use multus to create a pod with multiple network interfaces")
+
+	// 1. First create net-attach-def using bridgeCNI
+	nad_yaml := exutil.FixturePath("testdata", "net-attach-defs", "bridge-nad.yml")
+	g.By(fmt.Sprintf("calling oc create -f %s", nad_yaml))
+
+	err = oc.AsAdmin().Run("create").Args("-f", nad_yaml).Execute()
+	expectNoError(err)
+
+	oc.AsAdmin().Run("get").Args("network-attachment-definitions").Execute() // Print verification that it's come up
+
+	// 2. Then make a pod with the right annotation to use the net-attach-def
+	_ = launchTestMultusPod(f, podName)
+
+	// 3. Check that the pod is up
+	_, err = waitForTestMultusPod(f, ns, podName)
+	expectNoError(err)
+
+	// 4. Then need to inspect the pod and verify net1 interface exists.
+	output, err := oc.AsAdmin().Run("exec").Args("multus-test-pod", "--", "ip", "a", "show", "dev", "net1").Output()
+	expectNoError(err)
+	fmt.Println(output)
+	o.Expect(strings.Contains(output, "net1")).Should(o.Equal(true))
+
+	return nil
+}
+
+// launches a centos pod that sleeps
+func launchTestMultusPod(f *e2e.Framework, podName string) *kapiv1.Pod {
+	contName := fmt.Sprintf("%s-container", podName)
+	pod := &kapiv1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: podName,
+		},
+		Spec: kapiv1.PodSpec{
+			Containers: []kapiv1.Container{
+				{
+					Name:    contName,
+					Image:   image.LocationFor("docker.io/centos/httpd-24-centos8:latest"),
+					Command: []string{"/bin/bash", "-c", "trap : TERM INT; sleep infinity & wait"},
+					SecurityContext: &kapiv1.SecurityContext{
+						Capabilities: &kapiv1.Capabilities{
+							Add: []kapiv1.Capability{"NET_ADMIN"},
+						},
+					},
+				},
+			},
+			RestartPolicy: kapiv1.RestartPolicyNever,
+		},
+	}
+
+	pod.ObjectMeta.Annotations = map[string]string{
+		"k8s.v1.cni.cncf.io/networks": "bridge-nad",
+	}
+
+	_, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.Background(), pod, metav1.CreateOptions{})
+	expectNoError(err)
+	return pod
+}
+
+// waits till pod is ready
+func waitForTestMultusPod(f *e2e.Framework, namespace string, podName string) (string, error) {
```

Comment:
(this won't be needed if you use `CreateExecPodOrFail` because it does this for you)